bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -221,8 +221,7 @@ def cmake_enable_option(spec_requiremement):
             args.append("-DNRN_SANITIZERS=" + ",".join(self.spec.variants["sanitizers"].value))
         if "+mpi" in self.spec:
             args.append("-DNRN_ENABLE_MPI=ON")
-            if "~coreneuron" in self.spec:
-                args.append("-DNRN_ENABLE_MPI_DYNAMIC=ON")
+            args.append("-DNRN_ENABLE_MPI_DYNAMIC=ON")
```

Comment:
Could these not be unified on the Neuron side?

Reply:
@matz-e, it's a valid comment! Will consider that on the neuron side with cmake/recipe refactoring. I will merge this as we need this in the current BBP deployment to fix the issues like [BSD-347](https://bbpteam.epfl.ch/project/issues/browse/BSD-347).